cmd/broadcast/utils.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bufio"
+	"encoding/hex"
+	ida "github.com/harmony-one/libunison/internal/ida/coopcast"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const pubKeySize int = 20
+
+// Entry is a single config of a node.
+type Entry struct {
+	Sid     string
+	IP      string
+	TCPPort string
+	UDPPort string
+	PubKey  string
+	Role    string
+}
+
+// Config is a struct containing multiple Entry of all nodes.
+type Config struct {
```

Comment:
`s/Config/AllPeerConfig/`?

Also, a struct with a single eponymous member seems redundant.  Why not this instead?

    type AllPeerConfig []PeerConfig

Reply:
Actually, I think `Config` is okay, because the intention behind this struct is the all-inclusive set of configurations for this process; I guess you could simply rename the `config` member to `peerConfigs`.  On the other hand, configurable parameters (`flag.*` from `main()`) could be introduced here.